test/crypto/pycurl_test.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+  unicode_literals
+
+from unittest import TestCase
+
+from iota import TryteString
+from iota.crypto.pycurl import Curl
+
+
+class TestCurl(TestCase):
+    def test_correct_first(self):
```

Comment:
Could you add docstrings to each unit test, describing the situation it is testing?  The test names are a good start, but a little extra documentation describing the context for each test will go a long way.